Code-the-Dream-School/ffprac-team6
# Pp 44 implement crud for cards backend

Implemented CRUD operators on Backend.

I used JWT token on backend to receive userId.
according [documentation](https://next-auth.js.org/tutorials/securing-pages-and-api-routes#using-gettoken) 
Tokens you can use the getToken() helper to access the contents of the JWT without having to handle JWT decryption / verification yourself.
<img width="976" alt="image" src="https://github.com/Code-the-Dream-School/ffprac-team6/assets/106414157/d1d22d0f-b72a-40ec-b32a-9c6b325df21a">
 
So I didn't implement verification.

**How to test:**

After login go to the route and click add card:
<img width="420" alt="image" src="https://github.com/Code-the-Dream-School/ffprac-team6/assets/106414157/856e9742-76af-4e96-8a35-cb6dac4d2fbf">
than you should see card (I used hardcode card property due lack of create card form)
<img width="496" alt="image" src="https://github.com/Code-the-Dream-School/ffprac-team6/assets/106414157/6ff4895f-3c40-44af-b9b7-a1e964fd95f7">
click link edit card: 
<img width="137" alt="image" src="https://github.com/Code-the-Dream-School/ffprac-team6/assets/106414157/6dc9444c-526a-41e6-9245-c0cba01fea82">
you will be redirected on the single card route to edit it
<img width="682" alt="image" src="https://github.com/Code-the-Dream-School/ffprac-team6/assets/106414157/9b520d5d-5903-464f-9cdd-365c67e0144d">
click on "edit card" button 
the card name and card picture will be edited (I used hardcode card property due lack of edit card form)
<img width="560" alt="image" src="https://github.com/Code-the-Dream-School/ffprac-team6/assets/106414157/c604a3e3-d273-4cb1-bd51-4c6d7db9b184">
